pkg/dns/provider/changemodel.go

```diff
@@ -71,7 +71,7 @@ func (this *ChangeGroup) cleanup(logger logger.LogContext, model *ChangeModel) b
 	for _, s := range this.dnssets {
 		_, ok := model.applied[s.Name]
 		if !ok {
-			if s.IsOwnedBy(model.owners) {
```

Comment:
may be use `ownership`

Reply:
renamed interface to `Ownership`, variables to `ownership`